Formula/buildpulse-test-reporter.rb

```diff
@@ -0,0 +1,27 @@
+class BuildpulseTestReporter < Formula
+  desc "Connect your CI to BuildPulse to detect, track, and rank flaky tests"
+  homepage "https://buildpulse.io"
+  url "https://github.com/buildpulse/test-reporter/archive/refs/tags/v0.15.0.tar.gz"
+  sha256 "5119c189341bec12531fa85ac504a3c88b5ca08f090f2ee92d6cef667d66684d"
+  license "MIT"
+  head "https://github.com/buildpulse/test-reporter.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    goldflags = %W[
+      -s -w
+      -X main.Version=#{version}
+      -X main.Commit=#{tap.user}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags: goldflags), "./cmd/test-reporter"
+  end
+
+  test do
+    binary = bin/name
```

Comment:
Not really in favor of hiding the call like this. I get the reason but it's too much iyam.